### ISSUE TITLE
 Unhandled Exception: MissingPluginException(No implementation found for method scanBarcode on channel flutter_barcode_scanner)

### ISSUE THREAD

**Issue by MatteoColombo** [AmolGangadhare/flutter_barcode_scanner#27]
The scanner won't open and the exception is printed.

I don't know how to precisely reproduce it. It happened just one time in debug mode, while after compiling the apk and installing it on another devices it happens more frequently.
If I open the app, the barcode scanner works fine and then if I use other apps and after some time go back to the app, it randmoly stops working.

Test device:
 - Device: Oneplus 6
 - OS: Android 9

This are the logs printed on the device:
```
> 08-06 19:17:40.929 12759 23838 E flutter : [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method scanBarcode on channel flutter_barcode_scanner)
08-06 19:17:40.929 12759 23838 E flutter : #0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:314)
08-06 19:17:40.929 12759 23838 E flutter : <asynchronous suspension>
08-06 19:17:40.929 12759 23838 E flutter : #1      FlutterBarcodeScanner.scanBarcode (package:flutter_barcode_scanner/flutter_barcode_scanner.dart:34)
08-06 19:17:40.929 12759 23838 E flutter : <asynchronous suspension>
08-06 19:17:40.929 12759 23838 E flutter : #2      AddBookFloat._addBook (package:dima2018_colombo_troianiello/view/library-page/add-book-float.dart:22)
08-06 19:17:40.929 12759 23838 E flutter : <asynchronous suspension>
08-06 19:17:40.929 12759 23838 E flutter : #3      AddBookFloat.build.<anonymous closure> (package:dima2018_colombo_troianiello/view/library-page/add-book-float.dart:16)
08-06 19:17:40.929 12759 23838 E flutter : #4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:635)
08-06 19:17:40.929 12759 23838 E flutter : #5      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:711)
08-06 19:17:40.929 12759 23838 E flutter : #6      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182)
08-06 19:17:40.929 12759 23838 E flutter : #7      TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:365)
08-06 19:17:40.929 12759 23838 E flutter : #8      TapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:275)
08-06 19:17:40.929 12759 23838 E flutter : #9      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:455)
08-06 19:17:40.929 12759 23838 E flutter : #10     PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:75)
08-06 19:17:40.929 12759 23838 E flutter : #11     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:102)
08-06 19:17:40.929 12759 23838 E flutter : #12     _WidgetsFlutterBinding&BindingBase&GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:218)
08-06 19:17:40.929 12759 23838 E flutter : #13     _WidgetsFlutterBinding&BindingBase&GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:198)
08-06 19:17:40.929 12759 23838 E flutter : #14     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:156)
08-06 19:17:40.929 12759 23838 E flutter : #15     _WidgetsFlutterBinding&BindingBase&GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:102)
08-06 19:17:40.929 12759 23838 E flutter : #16     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:86)
08-06 19:17:40.929 12759 23838 E flutter : #17     _rootRunUnary (dart:async/zone.dart:1136)
08-06 19:17:40.929 12759 23838 E flutter : #18     _CustomZone.runUnary (dart:async/zone.dart:1029)
08-06 19:17:40.929 12759 23838 E flutter : #19     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931)
08-06 19:17:40.929 12759 23838 E flutter : #20     _invoke1 (dart:ui/hooks.dart:250)
08-06 19:17:40.929 12759 23838 E flutter : #21     _dispatchPointerDataPacket (dart:ui/hooks.dart:159)
08-06 19:17:40.929 12759 23838 E flutter : 
08-06 19:17:40.920 12759 12759 W 3.gpu   : type=1400 audit(0.0:78692): avc: denied { read } for name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=21589 scontext=u:r:untrusted_app:s0:c230,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
08-06 19:17:40.931 12759 23839 E libc    : Access denied finding property "vendor.debug.egl.swapinterval"
08-06 19:17:40.930 12759 12759 W 3.gpu   : type=1400 audit(0.0:78693): avc: denied { read } for name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=21589 scontext=u:r:untrusted_app:s0:c230,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
08-06 19:17:40.944 12759 23839 E libc    : Access denied finding property "vendor.debug.egl.swapinterval"
08-06 19:17:40.950 12759 12759 W 3.gpu   : type=1400 audit(0.0:78694): avc: denied { read } for name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=21589 scontext=u:r:untrusted_app:s0:c230,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
08-06 19:17:40.958 12759 23839 E libc    : Access denied finding property "vendor.debug.egl.swapinterval"
08-06 19:17:40.970 12759 12759 W 3.gpu   : type=1400 audit(0.0:78695): avc: denied { read } for name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=21589 scontext=u:r:untrusted_app:s0:c230,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
08-06 19:17:40.976 12759 23839 E libc    : Access denied finding property "vendor.debug.egl.swapinterval"
08-06 19:17:41.000 12759 12759 W 3.gpu   : type=1400 audit(0.0:78696): avc: denied { read } for name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=21589 scontext=u:r:untrusted_app:s0:c230,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
08-06 19:17:41.011 12759 23839 E libc    : Access denied finding property "vendor.debug.egl.swapinterval"
```
I have no problem opening the camera within the app even after this happens, it's only the barcode scanner that doesn't open.

**Comment by AmolGangadhare** [comment]
Duplicate of #7, please refer to this issue

**Comment by AmolGangadhare** [comment]
Closing this as it is resolved in #7
